cardano-node/src/Cardano/CLI/Tx/Submission.hs

```diff
@@ -8,18 +8,15 @@
 
 {-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
 
-module Cardano.Node.TxSubmission (
+module Cardano.CLI.Tx.Submission (
```

Comment:
See comment above about merging this with `Cardano.CLI.TX`.

Reply:
Again, see above response.